features/options/contract/version.feature

```diff
@@ -6,13 +6,6 @@ Feature: osrm-contract command line options: version
 
     Background:
         Given the profile "testbot"
-    
-    Scenario: osrm-contract - Version, short
```

Comment:
:+1: I'll restore this when I rename the flag. I was going off a suggestion from someone else that we just remove the short version.